docs/Formula-Cookbook.md

```diff
@@ -74,11 +74,11 @@ Homebrew will try to guess the formula’s name from its URL. If it fails to do
 
 ### Fill in the `homepage`
 
-**We don’t accept formulae without a [`homepage`](https://rubydoc.brew.sh/Formula#homepage%3D-class_method)!**
+**We don’t accept formulae without a [`homepage`](https://rubydoc.brew.sh/Formula#homepage-class_method)!**
```

Comment:
Note that this also makes the docs more correct, there never was a `homepage=` method, the setter was always `homepage "https://example.com"` (and similarly for the other `attr_rw` methods that were refactored in https://github.com/Homebrew/brew/pull/19359/files). 

(`%3D` is the URL encoding of `=`)